docker/nginx.conf

```diff
@@ -10,9 +10,8 @@ server {
 }
 
 server {
-  listen 443 ssl default_server;
-  server_name _;
-  root /var/www/html;
+  listen 443 ssl;
+  server_name filglancer-central.int.janelia.org fileglancer-central-dev.int.janelia.org;
```

Comment:
There's a typo in `filglancer` here. I guess you're mapping both prod and dev so that the same configuration can be used on both servers?